great_expectations/data_context/data_context/file_data_context.py

```diff
@@ -0,0 +1,14 @@
+from great_expectations.data_context.data_context.abstract_data_context import (
+    AbstractDataContext,
+)
+
+
+class FileDataContext(AbstractDataContext):
```

Comment:
@Shinnnyshinshin Same note here -- if we are going to have an `AbstractDataContext`, then it should really be "abstract" (i.e, having one or more `@abstractmethod`-decorated methods), and not able to run, until the abstract methods have concrete implementations.  Thanks.